### ISSUE TITLE
Terminal accessibility is broken?

### ISSUE THREAD

**Issue by rmwilliams2023** [microsoft/vscode#172394]
@meganrogge Type: <b>Bug</b>

The accessibility buffer is not behaving as before in today's insiders build. I cannot scroll through the output. Rather, all output seems to be read at once when attempting to tab back from the buffer to the terminal. If a regression has indeed occurred, it must be in one of the last two iterations.

VS Code version: Code - Insiders 1.75.0-insider (270c3b12906a88e9c67fc5ddef87be6aa3b75551, 2023-01-25T05:23:17.608Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: Yes

<details><summary>Extensions (6)</summary>

Extension|Author (truncated)|Version
---|---|---
cmake-tools|ms-|1.13.41
cpptools|ms-|1.14.0
cpptools-extension-pack|ms-|1.3.0
r|REd|2.7.0
multi-command|ryu|1.6.0
cmake|twx|0.0.17

(1 theme extensions excluded)

</details>

**Comment by meganrogge** [comment]
@rmwilliams2023 already fixed but it will not be available until the next insider's is released because Azure pipelines was down last night. see https://github.com/microsoft/vscode/pull/172276 for details
